charts/presto/templates/hive-server-statefulset.yaml

```diff
@@ -80,6 +80,10 @@ spec:
           mountPath: /hadoop/dfs/name
         - name: datanode-empty
           mountPath: /hadoop/dfs/data
+{{- if and .Values.spec.config.sharedVolume.enabled .Values.spec.config.sharedVolume.createPVC }}
```

Comment:
You don't need an and condition here, nor `.Values.spec.config.sharedVolume.createPVC`